samples/cs/FoundryLocal-01-SK-Chat/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿#pragma warning disable SKEXP0001, SKEXP0003, SKEXP0010, SKEXP0011, SKEXP0050, SKEXP0052
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.ChatCompletion;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+using System.Text;
+
+var model = "Phi-3.5-mini-instruct-cuda-gpu";
+var baseUrl = "http://localhost:5273/v1";
+var apiKey = "unused";
+
```

Comment:
Would it be possible to use the SDEK to get the `baseUrl` and `apiKey` in future releases these will be dynamically set.